simple_install/install.bat.in

```diff
@@ -0,0 +1,180 @@
+@echo off
+
+@rem This script requires the user to install Python 3.9 or higher. All other
+@rem requirements are downloaded as needed.
+
+@rem change to the script's directory
+PUSHD "%~dp0"
+
+set "no_cache_dir=--no-cache-dir"
+if "%1" == "use-cache" (
+    set "no_cache_dir="
+)
+
+exit /b
+
+echo ***** Installing InvokeAI.. *****
+@rem Config
+
+@rem this should be changed to the tagged release!
+@rem set INVOKE_AI_SRC=https://github.com/invoke-ai/InvokeAI/archive/main.zip
+set INVOKE_AI_SRC=https://github.com/invoke-ai/InvokeAI/archive/lstein-simple-install.zip
+set INSTRUCTIONS='<insert URL here>'
+set TROUBLESHOOTING='<insert URL here>'
+set MINIMUM_PYTHON_VERSION=3.9.0
+
+@rem ---------------------------- check Python version ---------------
+call python --version >.tmp1 2>.tmp2
+if "%ERRORLEVEL%" neq "0" (
+   echo Please install Python 3.9 or higher. See %INSTRUCTIONS% for details.
+   exit /b
+
+where /q virtualenv
+if %errorlevel% ne 0 (
+   echo Installing virtualenv support.
+   call pip install virtualenv
+   if %errorlevel% ne 0 (
+      echo The pip program failed to install virtualenv support. Please see %TROUBLESHOOTING% for help.
+      exit /b
+)
+   
+for /f "tokens=2" %%i in (<.tmp1) do set python_version=%%i
```

Comment:
Unfortunately, this is as far as I got tonight.
```
The system cannot find the file specified.
< was unexpected at this time.
```

I think there's a problem with `(<.tmp1)`, but I can not for the life of me find the correct syntax.

Reply:
Stop working on the bat script! I didn't realize you were toiling at it, and wrote the thing on the train coming back home this evening.

Reply:
All good, once I got to that point I realized that it's in an unfinished state. I didn't spend much time with it, no worries. Your new version looks like an impressive feat - I will test it out today in a VM